webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/index.js

```diff
@@ -94,17 +94,22 @@ const RegistrationCommandsPage = () => {
   // Plugins
   const [pluginValues, setPluginValues] = useState({});
 
-  const handlePluginValue = data => {
-    setPluginValues({ ...pluginValues, ...data });
-  };
+  const handlePluginValue = useCallback(data => {
+    setPluginValues(prevValues => ({ ...prevValues, ...data }));
+  }, []);
```

Comment:
The empty dep array here means
1. there actually aren't any dependencies that eslint complained about; and
2. `handlePluginValue` will always be the same function, and won't be redefined repeatedly.